docs/aerodromes/Melbourne.md

```diff
@@ -14,17 +14,31 @@
 | Melbourne ATIS        |                | 118.000         | YMML_ATIS                                |
 
 ## Airspace
+### Sunbury Corridor
 **ML ADC** is responsible for the Sunbury Corridor airspace as shown below. It is only available to Day VFR Helicopters.
 
 <figure markdown>
 ![Melbourne TCU Airspace Administration](img/MLTCUairspace.png){ width="500" }
   <figcaption>Melbourne TCU Airspace Administration</figcaption>
 </figure>
 
+Due to the close proximity of the airspace to the arrival and departure paths at YMML, controllers should be aware of surrounding traffic before issuing a clearance to helicopters.  If necessary, issue a clearance limit for separation, then instruct helicopters to report sighting and maintain own separation with other aircraft.  Alternatively, tower controllers can simulate visual separation provided no risk of collision exists and both aircraft remain in sight of the controller at all times.
+
+!!! example
+    **HEMS3:** "Melbourne Tower, HEMS3, approaching Sunbury Water Tank, A020, for Essendon via the Sunbury Corridor, request clearance"  
+    **ML ADC:** "HEMS3, cleared to Essendon, track Sunbury Corridor, not above A020, clearance limit Powerline Crossing"  
+    **HEMS3:** "Cleared to Essendon, track Sunbury Corridor, not above A020, clearance limit Powerline Crossing, HEMS3"  
+
+    **ML ADC:** "HEMS3, report sighting a Jetstar A320, 4nm final runway 16"  
+    **HEMS3:** "Traffic sighted, HEMS3"  
+    **ML ADC:** "HEMS3, cleared direct Melbourne, thence Essendon, pass behind that aircraft and maintain own separation"  
```

Comment:
Perhaps a little bit pedantic, but technically should be an additional check in there of "HEMS3, confirm you are able to maintain own separation with the Jetstar A320?"

Reply:
Does there also need to be a "cancel clearance limit"?

Reply:
You'd know better than me

Reply:
9.2.2.12 Cancelling clearance limit
When a clearance limit is cancelled, issue an onwards clearance specifying the level
and route to be flown from that point.

If we want to teach Clearance Limits, it'll need a bit more then...
"HEMS3, Cancel Clearance Limit, Pass Behind A320 4NM Final Runway 16, thence Cleared Melbourne, Essendon, 2000ft, Caution Wake Turbulence."

Reply:
Maybe I'll remove the clearance limit and just give the pass behind instruction, for simplicity in the example, and we can address clearance limits in controller skills separately?

Reply:
Concur